lnrpc/devrpc/dev.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+import "lightning.proto";
```

Comment:
Because of this, we'll also need the following snippet in `gen_protos.sh` to fix the CI error (after L57):
```
    if [[ "$package" == "devrpc" ]]; then
      manual_import="github.com/lightningnetwork/lnd/lnrpc"
    fi
```

Reply:
Thank you! Done.